beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -323,6 +323,7 @@ impl<E: EthSpec> PeerManager<E> {
     #[allow(clippy::mutable_key_type)]
     pub fn peers_discovered(&mut self, results: HashMap<Enr, Option<Instant>>) {
         let mut to_dial_peers = 0;
+        let results_count = results.len();
```

Comment:
nit: we are just using `results_count`  for the `if` conditional and the `debug!` log, I'd say it's faster to understand the code if we just use `results.len()` as there is one less layer of indirection.

Reply:
result is consumed at line 328, I think it's best to store the len in a variable than clone all ENRs

Reply:
right, thanks for explaining Lion :)